commands/combat.py

```diff
@@ -918,7 +903,7 @@ def func(self):
         client_width = self.client_width()
         caller = self.caller
         args = self.args
-        arts = Arts.objects.filter(characters=caller)
+        arts, base_arts, normals = filter_and_modify_arts(caller)
```

Comment:
might recommend changing the nomenclature for the variables so it's the same as in `filter_and_modify_arts` since the returned values are the modified versions of these things. e.g. `normals` -> `modified_normals`